go/vt/vtctl/reparentutil/emergency_reparenter.go

```diff
@@ -594,10 +606,10 @@ func (erp *EmergencyReparenter) reparentReplicas(
 
 	primaryErr := handlePrimary(topoproto.TabletAliasString(newPrimaryTablet.Alias), newPrimaryTablet)
 	if primaryErr != nil {
-		erp.logger.Warningf("primary failed to PopulateReparentJournal")
+		erp.logger.Warningf("primary failed to promote the new primary")
```

Comment:
This message doesn't make sense. Maybe it should be just `failed to promote new primary`? And if so, is it worth printing out the alias of the new primary? Which would end up being something like `failed to promote %s to primary` with the tablet alias plugged in.

Reply:
Fixed